Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,18 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes.git", :tag => "v2.2.1"
```

Comment:
Just use an `archive` tarball unless there's a good reason not to.

Reply:
Thanks.
Fixed in 80bc91c